docs/ways-of-work/plan/cloud-workspace/private-editor/implementation-plan.md

```
@@ -0,0 +1,510 @@
+2. Verify simfile ownership via Supabase query
+3. Upload DTX content to R2 at key `{simfileID}/.draft/{difficulty}.dtx`
+4. Upsert `simfile_drafts` record with metadata + `last_saved_at = now()`
+5. Return success with timestamp
+
+#### 2. `GET /api/simFile/loadDraft/[simfileID]`
+
+**Purpose**: Load draft data for a simfile (all difficulties or specific one).
+
+**Request**:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)